src/test/compile-fail/regions-escape-loop-via-vec.rs

```diff
@@ -12,10 +12,18 @@
 fn broken() {
     let mut x = 3;
     let mut _y = vec!(&mut x);
+    //~^ NOTE reference must be valid for the block suffix following statement 1 at 14:30...
+    //~| NOTE borrow of `x` occurs here
+    //~| NOTE borrow of `x` occurs here
+    //~| NOTE borrow of `x` occurs here
```

Comment:
Yup, that's right.  It actually shows up that many times in the test since there are multiple tests that will label that same span.